src/Symfony/Component/HttpKernel/CacheWarmer/CacheWarmerAggregate.php

```diff
@@ -21,12 +21,16 @@
 class CacheWarmerAggregate implements CacheWarmerInterface
 {
     private $warmers;
+    private $debug;
+    private $containerPathPrefix;
     private $optionalsEnabled = false;
     private $onlyOptionalsEnabled = false;
 
-    public function __construct(iterable $warmers = array())
+    public function __construct(iterable $warmers = array(), bool $debug = false, string $containerPathPrefix = null)
     {
         $this->warmers = $warmers;
+        $this->debug = $debug;
+        $this->containerPathPrefix = $containerPathPrefix;
```

Comment:
I'd suggest to pass the path of the deprecation file instead, not only the prefix

Reply:
`$containerPathPrefix` became `$deprecationLogsFilepath` with the right argument.